rotkehlchen/rotkehlchen.py

```diff
@@ -168,6 +169,14 @@ def __init__(self, args: argparse.Namespace) -> None:
         self.shutdown_event = gevent.event.Event()
         self.migration_manager = DataMigrationManager(self)
 
+    def kill_running_tx_query_tasks(self) -> None:
+        """Attempts to kill greenlets tasks querying transactions."""
```

Comment:
Why does it "attempt"? Are there any scenarios when it fails to kill the greenlets?

Reply:
If no greenlet is running at that point, perhaps the docstring can be better.